src/main/resources/com/atlassian/bitbucket/jenkins/internal/jenkins/oauth/token/OAuthTokenConfiguration/index.properties

```diff
@@ -0,0 +1,5 @@
+bitbucket.oauth.token.consumer.name=Consumer Name
+bitbucket.oauth.token.creation.date=Creation Date
+bitbucket.oauth.token.status=Token Status
+bitbucket.oauth.token.action.button=Action
+bitbucket.oauth.token.creation.date.value={0,date,yyyy-mm-dd hh:mm}
```

Comment:
Filed [JENKINS-66372](https://issues.jenkins.io/browse/JENKINS-66372) for the repeated `mm`.